apps/frontend/src/app/3_organisms/ZeroLocForm/components/AdjustCreditLine.tsx

```diff
@@ -33,6 +33,7 @@ type AdjustCreditLineProps = {
   onSubmit: (value: CreditLineSubmitValue) => void;
   rbtcPrice: Decimal;
   borrowingRate: Decimal;
+  maxBorrowingFeeRate: Decimal;
```

Comment:
No need to pass it as a prop since it's fetched in a hook, just call the hook in the component.

The same applies to `OpenCreditLine`.